assets/javascripts/discourse/admin/models/ai-tool.js

```diff
@@ -32,8 +32,10 @@ export default class AiTool extends RestModel {
 
         if (parameter.enum && parameter.enum.length) {
           parameter.enum = new TrackedArray(parameter.enum);
+          parameter.isEnum = true;
         } else {
           parameter.enum = null;
+          parameter.isEnum = false;
```

Comment:
I think we need this to be opaque to rest model cause we don't want to send it to the server